### DISCUSSION TITLE
Lodestar Standup 2024-14 (April 9, 2024)

### DISCUSSION THREAD

**Discussion by philknows** [ChainSafe/lodestar#6653] [Standups]
### Meeting Info
[Apr 9, 2024, 14:00-15:00 UTC](https://savvytime.com/converter/utc-to-germany-berlin-united-kingdom-london-ny-new-york-city-ca-san-francisco-china-shanghai-japan-tokyo-australia-sydney/apr-9-2024/2pm)
Zoom: shared on #lodestar-general ChainSafe Discord channel at 14:00 UTC

### Agenda
- Planning Items
  - v1.18.0 beta
- Team Updates
- Open Discussion/Closing Remarks

### Instructions
Please post here any topics to be added for Standup discussions. Examples of agenda items are issues, discussions, proposals which require feedback, consensus from the team and/or community members.

**Comment by twoeths** [comment]
we failed to take heap dump of network thread on our infrastructure but @nflaig could, would like to specify different specifications in order to do that in our infrastructure:
- memory requirement
- service deployment vs docker deployment

**Comment by twoeths** [comment]
looks like we're almost done or in-progress for https://notes.ethereum.org/@ethpandaops/pectra-devnet-0 except for https://eips.ethereum.org/EIPS/eip-7549

**Reply by philknows** [reply]
Thinking we should setup a day here to go through what was done for these features so far so everybody has an idea of the state of the code going into the Interop. Pitching for May 9. Will discuss further on call.

**Comment by nflaig** [comment]
We should probably make a final decision on the peer count and if we wanna wait with the release for it. I increasingly see users asking for this and bump the count manually anyways, and I am running with 100 peers on mainnet for a month now without any issues but notably better effectiveness.

So us not bumping the default just means we don't do sufficient testing with 100 peers

Fits also in the discussion that "defaults are sticky", when it comes to peer count, this does not seem to be the case at all

**Comment by philknows** [comment]
Closing for #6671
